tests/foreman/cli/test_host.py

```diff
@@ -2309,7 +2301,7 @@ def test_positive_auto_attach(request, module_host_subscription, host_subscripti
     host = Host.info({'name': module_host_subscription.client.hostname})
     module_host_subscription._register_client(activation_key=activation_key)
     Host.subscription_auto_attach({'host-id': host['id']})
-    result = module_host_subscription.enable_repo(module_host_subscription.client.repository_id)
+    result = module_host_subscription.client.enable_repo(module_host_subscription.repository_id)
```

Comment:
I don't think this method actually returns anything, unless there is another `enable_repo` defined somewhere?

https://github.com/SatelliteQE/robottelo/blob/67c8165fa670e36584fdd9f06ffb0af5aea51b80/robottelo/hosts.py#L248

It can be easily updated to return the result of `self.execute` so that you can assert against the status.

Reply:
Note this assertion was the only failure in Robottelo-Runner checks. 

`AttributeError: 'NoneType' object has no attribute 'status'`

Reply:
yes I forget to remove it from all occurrences, it's fixed now.

Reply:
I believe it's better to: `subscription manager repos --enabled` before it's added and afterwards.